contracts/test/hardhat/contracts/FleekERC721/collection-roles.t.ts

```diff
@@ -175,4 +175,39 @@ describe('FleekERC721.CollectionRoles', () => {
       contract.revokeCollectionRole(CollectionRoles.Owner, owner.address)
     ).to.be.revertedWithCustomError(contract, Errors.MustHaveAtLeastOneOwner);
   });
+
+  it('should not be able to verify access point if not verifier', async () => {
+    const { owner, contract, otherAccount } = fixture;
+
+    await contract.mint(
+      owner.address,
+      TestConstants.MintParams.name,
+      TestConstants.MintParams.description,
+      TestConstants.MintParams.externalUrl,
+      TestConstants.MintParams.ens,
+      TestConstants.MintParams.commitHash,
+      TestConstants.MintParams.gitRepository,
+      TestConstants.MintParams.logo,
+      TestConstants.MintParams.color,
+      TestConstants.MintParams.accessPointAutoApprovalSettings
+    );
+
+    await contract.addAccessPoint(0, 'random.com');
+
+    await expect(
+      contract
+        .connect(otherAccount)
+        .setAccessPointContentVerify('random.com', true)
+    )
+      .to.be.revertedWithCustomError(contract, Errors.MustHaveCollectionRole)
```

Comment:
Maybe we can expect rever with calling it by token owner too to make sure they should never be able to set it.

Reply:
Changed the token owner to be `otherAccount` since `owner` also has the verifier role